docs/src/playground/StocksPage.stories.tsx

```diff
@@ -5,14 +5,6 @@ import ExamplePage from './ExamplePage';
 export default {
   title: 'Playground',
   component: ExamplePage,
-  decorators: [
```

Comment:
I removed this decorator since the padding it's trying to override will be removed in #187 .